examples/ocaml_examples.ml

```diff
@@ -41,5 +41,5 @@ let _ =
           ~body:(Fetch.BodyInit.make
                    (Js.Json.stringify (Js.Json.object_ payload)))
           ~headers:(Fetch.HeadersInit.make
-                      ([%bs.obj { Content-Type = "application/json" }])) ()))
-      |> (then_ Fetch.Response.json)
\ No newline at end of file
+                      ([%bs.obj { contentType = "application/json" }])) ()))
```

Comment:
This doesn't mean the same thing. `fetch` expects a js object with header names as they should appear in the http request. The Reason syntax allows invalid identifiers to be sued as object keys, but OCaml does not, so the translation is wrong (or rather, there is no valid translation for this).

`fetch`, and `bs-fetch`, supposedly also supports an array of pairs, but many implementations don't. We should probably add a `fromDict` function too. Could you make an issue about this? And if you need this to compile, just comment it out with a note.

Reply:
alright!